packages/canon/css/styles.css

```diff
@@ -9318,6 +9318,36 @@
   height: 1.5rem;
 }
 
+.canon-DataTableRoot {
+  gap: var(--canon-space-3);
+  flex-direction: column;
+  display: flex;
+}
+
+.canon-DataTablePagination {
+  padding-top: var(--canon-space-5);
+  justify-content: space-between;
+  align-items: center;
+  display: flex;
+}
+
+.canon-DataTablePagination--left {
+  justify-content: space-between;
+  align-items: center;
+  display: flex;
+}
+
+.canon-DataTablePagination--right {
+  justify-content: space-between;
+  align-items: center;
+  gap: var(--canon-space-2);
+  display: flex;
+}
+
+.canon-DataTablePagination--select {
```

Comment:
just noting that this is a third duplication, please ignore me if thats expected

Reply:
No worries. I just answered to it in the above questions.